.pre-commit-config.yaml

```diff
@@ -3,22 +3,49 @@ repos:
     hooks:
       - id: web-js-lint-check
         name: Web JS Lint Check
-        entry: bash -c 'make -j$(nproc) -C webui/react check-eslint check-prettier-js'
+        entry: >
+          bash -c
+          '
+          cd webui/react &&
+          FILES=$(
+            git diff --name-only --staged --relative src/**/*.js src/**/*.jsx src/**/*.ts src/**/*.tsx |
+            tr "\n" " "
+          ) &&
+          make prettier ARGS="-- -c $FILES" &&
+          make eslint ARGS="$FILES"
```

Comment:
I wanted to combine these 2 make commands for parallel run but I got errors somehow. Any idea? I did `make prettier ARGS="-- -c $FILES" eslint ARGS="$FILES"`

Reply:
Sounds to me like make just isn't the right tool for this job, why not call the npm stuff directly?

Reply:
I wanted to use `make` to run commands parallel with `-j`, and I thought its more consistent to use `make`. However, I'm not against using `npm`.